src/importers/nordpassCsvImporter.ts

```diff
@@ -0,0 +1,139 @@
+                    cipher.card.cardholderName = this.getValueOrDefault(record.cardholdername);
+                    cipher.card.number = this.getValueOrDefault(record.cardnumber);
+                    cipher.card.code = this.getValueOrDefault(record.cvc);
+                    cipher.card.brand = this.getCardBrand(cipher.card.number);
+                    this.setCardExpiration(cipher, record.expirydate);
+                    break;
+
+                case CipherType.Identity:
+                    cipher.type = CipherType.Identity;
+
+                    this.processName(cipher, this.getValueOrDefault(record.full_name));
+                    cipher.identity.address1 = this.getValueOrDefault(record.address1);
+                    cipher.identity.address2 = this.getValueOrDefault(record.address2);
+                    cipher.identity.city = this.getValueOrDefault(record.city);
+                    cipher.identity.state = this.getValueOrDefault(record.state);
+                    cipher.identity.postalCode = this.getValueOrDefault(record.zipcode);
+                    cipher.identity.country = this.getValueOrDefault(record.country);
+                    if (cipher.identity.country != null) {
+                        cipher.identity.country = cipher.identity.country.toUpperCase();
+                    }
+                    cipher.identity.email = this.getValueOrDefault(record.email);
+                    cipher.identity.phone = this.getValueOrDefault(record.phone_number);
+                    break;
+                case CipherType.SecureNote:
+                    cipher.type = CipherType.SecureNote;
+                    cipher.secureNote.type = SecureNoteType.Generic;
+                    break;
+                default:
+                    break;
+            }
+
+            this.cleanupCipher(cipher);
+            result.ciphers.push(cipher);
+        });
+
+        if (this.organization) {
+            this.moveFoldersToCollections(result);
+        }
+
+        result.success = true;
+        return Promise.resolve(result);
+    }
+
+    private evaluateType(record: nodePassCsvParsed): CipherType {
+
+        if (!this.isNullOrWhitespace(record.username)) return CipherType.Login;
+
+        if (!this.isNullOrWhitespace(record.cardnumber)) return CipherType.Card;
+
+        if (!this.isNullOrWhitespace(record.full_name)) return CipherType.Identity;
+
+        if (!this.isNullOrWhitespace(record.note)) return CipherType.SecureNote;
+
+        return undefined;
+    }
+
+    private processName(cipher: CipherView, fullName: string) {
+
+        if (this.isNullOrWhitespace(fullName)) return;
+
+        const nameParts = fullName.split(' ');
+        if (nameParts.length > 0) {
+            cipher.identity.firstName = this.getValueOrDefault(nameParts[0]);
+        }
+        if (nameParts.length === 2) {
+            cipher.identity.lastName = this.getValueOrDefault(nameParts[1]);
+        } else if (nameParts.length === 3) {
```

Comment:
please consider `} else if (nameParts.length >= 3) {`, simply because of names which may include suffixes, titles or other info, `Gerald Edmond Richardson Jr. III`, we would not parse name in this instance when we could at least capture `Edmond` and `Richardson`; or consider re-joining all name parts past the `[2]`nd index with that value, concatenating the final last name/surname result (e.g. `Edmond` (middle) and `Richardson Jr. III` (lastname)).

Reply:
@cscharf Good point and I've added support to parse more complex names and wrote tests to verify.